apps/dotcom/client/src/tla/components/TlaSidebar/TlaSidebar.tsx

```diff
@@ -171,6 +169,11 @@ function TlaSidebarRecentFiles() {
 		},
 		[app]
 	)
+	const sidebarContentRef = useRef<HTMLDivElement>(null)
+
+	useEffect(() => {
+		sidebarContentRef.current?.scrollTo({ top: 0 })
```

Comment:
worth considering adding `behavior: smooth` if you like

Reply:
Added that (had to rework the logic a bit so that remote changes would not trigger the scroll).